src/app/app.component.ts

```diff
@@ -1,10 +1,9 @@
 import { Component } from '@angular/core';
-
```

Comment:
Please do not remove space here, We usually need a space between imports and component declaration. 
It is more readable this way.

Reply:
There is no space there on my screen?!